apps/dashboard/src/model-assessment-forecasting/App.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ITheme } from "@fluentui/react";
+import { HelpMessageDict } from "@responsible-ai/error-analysis";
+import { Language } from "@responsible-ai/localization";
+import {
+  ModelAssessmentDashboard,
+  IModelAssessmentDashboardProps,
+  IModelAssessmentData
+} from "@responsible-ai/model-assessment";
+import React from "react";
+
+interface IAppProps extends IModelAssessmentData {
+  theme: ITheme;
+  language: Language;
+  version: 1 | 2;
+  classDimension?: 1 | 2 | 3;
+  featureFlights?: string[];
+}
+
+export class App extends React.Component<IAppProps> {
+  private messages: HelpMessageDict = {
+    LocalExpAndTestReq: [{ displayText: "LocalExpAndTestReq", format: "text" }],
+    LocalOrGlobalAndTestReq: [
+      { displayText: "LocalOrGlobalAndTestReq", format: "text" }
+    ],
+    PredictorReq: [{ displayText: "PredictorReq", format: "text" }],
+    TestReq: [{ displayText: "TestReq", format: "text" }]
+  };
+
+  public render(): React.ReactNode {
+    if (this.props.modelExplanationData) {
+      for (const exp of this.props.modelExplanationData) {
```

Comment:
nit: this if is not needed right ? you could just iterate through this.props.modelExplanationData which if it's not defined wouldn't do anything

Reply:
This is indeed a strange way of writing this. Let me adjust that....